app/ClientComponent.tsx

```diff
@@ -1,6 +1,6 @@
 "use client";
 
-import { myTest } from "~/mylib";
+import { myTest } from "mylib";
```

Comment:
q: does this work? to the best of my knowledge this should be written like this:
```suggestion
import { myTest } from "#mylib";
```